icu4c/source/data/coll/es__TRADITIONAL.txt

```diff
@@ -1,7 +0,0 @@
-﻿// © 2016 and later: Unicode, Inc. and others.
-// License & terms of use: http://www.unicode.org/copyright.html#License
-es__TRADITIONAL{
```

Comment:
Why did this file get deleted?
Also same for `de__PHONEBOOK.txt` as well?

Reply:
This might be a mistake, maybe by the cleanup code. My tool won't generate these (it cannot generate files with invalid locale names). I think we decided to leave these manually maintained, and they are not mentioned in my anlysis:
https://docs.google.com/spreadsheets/d/10UqiRFEXVgFygSchF7ZhWcz5pC0_gqAE5sgPGIB36iM/edit#gid=474876707
So at this time I'm suspecting I'm deleting it by mistake.

There is a way to specify files not to touch in the Ant build-icu-data.xml file which might need updated.

Reply:
Hmm, it should be keeping them since they are explicitly mentioned in:
https://github.com/unicode-org/icu/blob/master/tools/cldr/cldr-to-icu/build-icu-data.xml#L350
So if it is the tool that's deleting then, it's an error.

I suspect it's actually that Peter ran an additional (and as far as these data files are concerned unnecessary) cleanup step before running the new tooling (I saw something in the updated instructions). The new tool handles these directories with the explicit assumption that some files exist and get preserved (i.e. they are now only manually maintained).

Reply:
I did run ant clean in icu4c/source/data before running the new tooling. How else to handle e.g. regional locales that get removed, or that have no data in certain categories? Such files get removed in every integration. If we just preserve all existing files that are not overwritten by the new data we will end up with tons of cruft. I was assuming the tooling would generate all necessary stubs of the sort under consideration here, the old tooling did.